fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1933,11 +1933,18 @@ void SimulationConfig::setReplicationType(const TestConfig& testConfig) {
 			// FIXME: log replicas must be more than storage replicas because otherwise better master exists will not
 			// recognize it needs to change dcs
 			int replication_factor = deterministicRandom()->randomInt(storage_servers, generateFearless ? 4 : 5);
-			int anti_quorum = deterministicRandom()->randomInt(
-			    0,
-			    (replication_factor / 2) +
-			        1); // The anti quorum cannot be more than half of the replication factor, or the
-			            // log system will continue to accept commits when a recovery is impossible
+			// Version vector is an experimental feature that does not support logWriteAntiQuorum
+			// feature. Disable logWriteAntiQuorum (when version vector is enabled) in simulation
+			// tests for now.
+			// @todo extend version vector to support logWriteAntiQuorum feature.
```

Comment:
nit - all-caps TODO seems to be the convention. I don't think you disabling it just for simulation tests ?
```
 % git grep TODO|wc -l
     756
 % git grep '@todo'|wc -l
       7
```

Reply:
I am mainly using the Doxygen (https://www.doxygen.nl/) "todo" tag intended for this. Can use "TODO" - no issues.

Reply:
> Are you disabling it just for simulation tests ?

Correct.